Build error : use 'template' keyword to treat 'vector_index_set_t' as a dependent template

Error when building gysela : 

```
/Users/dasari/Desktop/gysela/gyselalibxx/src/math_tools/l_norm_tools.hpp:47:50: error: use 'template' keyword to treat 'vector_index_set_t' as a dependent template name
  47 |   using index_set = typename DVector<Tags...>::vector_index_set_t<0>;
   |                          ^
   |                          template

```
Built with : -DCMAKE_BUILD_TYPE=Release -DBLA_PREFER_PKGCONFIG=ON

--- a/src/math_tools/l_norm_tools.hpp
+++ b/src/math_tools/l_norm_tools.hpp
@@ -44,7 +44,7 @@ KOKKOS_FUNCTION double norm_inf(ddc::Coordinate<Tags...> coord)
 template <class... Tags>
 KOKKOS_FUNCTION double norm_inf(DVector<Tags...> vec)
 {
-    using index_set = typename DVector<Tags...>::vector_index_set_t<0>;
+    using index_set = typename DVector<Tags...>::template vector_index_set_t<0>;
     static_assert(
             std::is_same_v<index_set, vector_index_set_dual_t<index_set>>,
             "Mapping is needed to calculate norm_inf on a non-orthonormal coordinate system");

